playwright/tests/a11y.spec.js

```diff
@@ -1,66 +1,145 @@
+  'http://www.va.gov/education/download-letters/letters/',
+  'http://www.va.gov/health-care/appointment-pre-check-in/',
+  'http://www.va.gov/resources/search/',
+  'http://www.va.gov/va-payment-history/payments/',
+  'http://www.va.gov/records/get-veteran-id-cards/apply/',
+  'http://www.va.gov/health-care/order-hearing-aid-or-CPAP-supplies-form/',
+  'http://www.va.gov/education/gi-bill/post-9-11/ch-33-benefit/status/',
+  'http://www.va.gov/housing-assistance/home-loans/request-coe-form-26-1880/',
+  'http://www.va.gov/records/download-va-letters/letters/',
+  'http://www.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/',
+  'http://www.va.gov/burials-memorials/memorial-items/presidential-memorial-certificates/request-certificate-form-40-0247/',
+  'http://www.va.gov/my-health/order-hearing-aid-or-cpap-supplies-form/',
+  'http://www.va.gov/education/other-va-education-benefits/veteran-rapid-retraining-assistance/apply-for-vrrap-form-22-1990s/',
+  'http://www.va.gov/track-claims/',
+  'http://www.va.gov/burials-and-memorials-v2/application/530/',
+  'http://www.va.gov/profile/',
+  'http://www.va.gov/my-va/',
+  'http://www.va.gov/disability/view-disability-rating/rating/',
+  'http://www.va.gov/view-change-dependents/view/',
+  'http://www.va.gov/supporting-forms-for-claims/substitute-claimant-form-21P-0847/',
+  'http://www.va.gov/supporting-forms-for-claims/alternate-signer-form-21-0972/',
+  'http://www.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-25-8832/',
+]
 
-test.describe('Accessibility Tests', async () => {
+test.describe(`Accessibility Site Scan`, async () => {
   test.setTimeout(4200000)
 
-  const pages = await getSitemapLocations(
-    process.env.BASE_URL || 'http://127.0.0.1:8001'
-  )
+  test(`The sitemap should contain accessible pages`, async ({
+    page,
+    browserName,
+    // makeAxeBuilder,
+  }, testInfo) => {
+    const viewportSize = page.viewportSize()
+    let scanResults = []
+    let failedPages = []
+    let pages = await getSitemapLocations(baseUrl)
 
-  let pageSegments
+    if (segmentNumber !== 0) {
+      pages = splitArray(pages, totalSegments)[segmentNumber - 1]
+    }
 
-  test.beforeAll(async () => {
-    pageSegments = await splitPagesIntoSegments(pages, 5)
-  })
+    console.log('number of pages in segment', pages.length)
+    console.log('browser name:', browserName)
+    console.log('viewport size:', viewportSize)
+
+    for (const pageUrl of pages) {
+      if (excludedPages.includes(pageUrl)) {
+        console.log('skipping scan of excluded page:', pageUrl)
+        continue
+      }
+
+      try {
+        console.log('testing page:', pageUrl)
+        await page.goto(pageUrl)
+
+        // @todo The shared "makeAxeBuilder" never reports errors for whatever reason so not using for now.
+        // const accessibilityScanResults = await makeAxeBuilder({ page }).analyze()
```

Comment:
just confirming that this is still true for you? it has reported errors previously in the individual spec tests

Reply:
I added the fixture back and the report shows scan results. So the fixture looks to be working.